tools/python/boutiques/schema/descriptor.schema.json

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
Ya, I think that might be because there is a symbolic link with that original filename/path, so the history got confused. I'll try to fix it